sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -2283,7 +2286,12 @@ class Analyzer(override val catalogManager: CatalogManager)
                 messageParameters = Map(
                   "class" -> other.getClass.getCanonicalName))
               // We don't support persistent high-order functions yet.
```

Comment:
Can we move this comment one line before `Seq("system.builtin", "system.session")`? It explains why current database is not in the search path

Reply:
done